CODE_OF_CONDUCT.md

```diff
@@ -71,6 +71,27 @@ should not use sexualized images, activities, or other material. Booth staff
 (including volunteers) should not use sexualized clothing/uniforms/costumes,
 or otherwise create a sexualized environment.
 
+## Examples of Inappropriate Behavior</h2>
+
+- Harassment of any participants in any form
+- Deliberate intimidation, stalking, or following
+- Logging or taking screenshots of online activity for harassment purposes
+- Publishing others’ private information, such as a physical or electronic address, without explicit permission
+- Violent threats or language directed against another person
+- Incitement of violence or harassment towards any individual, including encouraging a person to commit suicide or to engage in self-harm
+- Creating additional online accounts in order to harass another person or circumvent a ban
+- Sexual language and imagery in online communities or in any conference venue, including talks
+- Insults, put downs, or jokes that are based upon stereotypes, that are exclusionary, or that hold others up for ridicule
+- Excessive swearing
+- Unwelcome sexual attention or advances
+- Unwelcome physical contact, including simulated physical contact (eg, textual descriptions like “hug” or “backrub”) without consent or after a request to stop
+- Pattern of inappropriate social contact, such as requesting/assuming inappropriate levels of intimacy with others
+- Sustained disruption of online community discussions, in-person presentations, or other in-person events
+- Continued one-on-one communication after requests to cease
+- Other conduct that is inappropriate for a professional audience including people of many different backgrounds
+
+Community members asked to stop any inappropriate behavior are expected to comply immediately.
+
```

Comment:
We should probably include a mention that this list is not an exhaustive list of all inappropriate behavior that is unacceptable at DjangoCon US.